clients/src/main/resources/common/message/ShareGroupHeartbeatRequest.json

```diff
@@ -33,9 +33,14 @@
       "about": "The current member epoch; 0 to join the group; -1 to leave the group." },
     { "name": "RackId", "type": "string", "versions": "0+",  "nullableVersions": "0+", "default": "null",
       "about": "null if not provided or if it didn't change since the last heartbeat; the rack ID of consumer otherwise." },
-    { "name": "RebalanceTimeoutMs", "type": "int32", "versions": "0+", "default": -1,
-      "about": "-1 if it didn't change since the last heartbeat; the maximum time in milliseconds that the coordinator will wait on the member to revoke its partitions otherwise." },
     { "name": "SubscribedTopicNames", "type": "[]string", "versions": "0+", "nullableVersions": "0+", "default": "null",
-      "about": "null if it didn't change since the last heartbeat; the subscribed topic names otherwise." }
+      "about": "null if it didn't change since the last heartbeat; the subscribed topic names otherwise." },
+    { "name": "TopicPartitions", "type": "[]TopicPartitions", "versions": "0+", "nullableVersions": "0+", "default": "null",
+      "about": "null if it didn't change since the last heartbeat; the partitions owned by the member.", "fields": [
+      { "name": "TopicId", "type": "uuid", "versions": "0+",
+        "about": "The topic ID." },
+      { "name": "Partitions", "type": "[]int32", "versions": "0+",
+        "about": "The partitions." }
+    ]}
```

Comment:
Question: shall we fill this data as well in group metadata manager?

Reply:
Not yet. The KIP has gone back on this again. Once we have approval, let's do a final refresh of all structures.